Update backup registry secret name (#3054)
codenvy/che-docs: modules/administration-guide/pages/devworkspace-backup-regular-oci-registry.adoc

    backupCronJob:
      enable: true
      registry:

>         authSecret: devworkspace-backup-registry-auth

Reviewer: The PR description says this fixed name is a workaround for a bug where the restore init container cannot access the pull secret. Is this the permanent intended behavior, or should the docs note this as a known limitation that may change?

Author: It's right now a known limitation, which we will likely change for the next release

        path: quay.io/my-company-org
      schedule: '0 */4 * * *'
    imagePullPolicy: Always
----
<1> For Red Hat OpenShift, the default installation namespace for the {devworkspace} operator is `openshift-operators`. See the xref:devworkspace-operator.adoc[{devworkspace} operator overview].

> The `authSecret` must be named `devworkspace-backup-registry-auth` and point to a real {kubernetes} Secret of type `kubernetes.io/dockerconfigjson` containing credentials to access the registry.

Reviewer: ```suggestion
The `authSecret` must be named `devworkspace-backup-registry-auth`. It must reference a {kubernetes} Secret of type `kubernetes.io/dockerconfigjson` that contains credentials to access the registry.
```

The secret should be created in the installation {namespace} for the {devworkspace} operator.

To create one, you can use the following command:

[source,shell,subs="+attributes,+quotes"]
----

> kubectl create secret docker-registry devworkspace-backup-registry-auth --from-file=config.json

Reviewer: Pre-existing: `kubectl` hardcoded. Use `{orch-cli}` for downstream compatibility (same on line 50).

```suggestion
{orch-cli} create secret docker-registry devworkspace-backup-registry-auth --from-file=config.json
```
